packages/components/src/components/as-time-series-widget/as-time-series-widget.tsx

```diff
@@ -267,13 +274,17 @@ export class TimeSeriesWidget {
     if (sameData(newData, oldData)) {
       return;
     } else {
-      this._data = prepareData(newData);
+      this._data = newData;
```

Comment:
This is great and probably improves the performance a bit :clap:

The question that remains is, do we actually need a this._data then? It's not a huge overhead (just another ref to the same data), but I think we can remove it for simplicity's sake.